src/main/java/it/pagopa/transactions/controllers/v1/TransactionsController.java

```diff
@@ -51,12 +53,15 @@ public class TransactionsController implements TransactionsApi {
     @Autowired
     private UpdateTransactionStatusTracerUtils updateTransactionStatusTracerUtils;
 
+    @Autowired
+    private OpenTelemetryUtils openTelemetryUtils;
+
     @ExceptionHandler(
         {
                 CallNotPermittedException.class
         }
     )
-    public Mono<ResponseEntity<ProblemJsonDto>> openStateHandler() {
+    public Mono<ResponseEntity<ProblemJsonDto>> openStateHandler(CallNotPermittedException error) {
```

Comment:
please add exception also to error log in order to log stacktrace into logs

Reply:
Solved by https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/502/commits/9549d6c61c741e1c1eda455abc490570a15a12ec